test/acceptance/integration/Rules/Cards/card_create_rules.feature

```diff
@@ -201,7 +201,7 @@ Feature: Append a new rule in Perseo manager using cards from portal
     | test1300  | 34       | temper_23      | 12       | SendSmsMibAction | ${device_latitude}${device_longitude}${measure.value} | 123456789       |
     | test1300  | 34       | temper_23      | 0        | updateAttribute  | DANGER                                                | ALARM           |
 
-  @not_updated_card_interval_not_allowed @BUG_53
+  @not_updated_card_interval_not_allowed @BUG_53 @skip
```

Comment:
Normally, the skip tag is used in scenarios that still are not developed, but the testcases are created or in cases that the scenario is failing and it is not necessary to execute (because the fail is not fix yet). We always reference the associated defect, in this case @BUG_53.The scenario is skipped only if the tag "skip" is invoked:
     Ex: lettuce_tools -tg=-skip